perllib/FixMyStreet/Cobrand/Gloucester.pm

```diff
@@ -109,6 +109,21 @@ sub categories_restriction {
 
 =back
 
+=head2 post_report_report_problem_link
```

Comment:
What does this do exactly? I'm not clear on what links it affects.

Reply:
This is used (only) by the `templates/web/base/navigation/_report.html` template to override the link shown in the navbar at this point. Looks like it was done for BANES, I discovered this when looking to find out where the navbar "Report another problem here"  link was coming from.